test/test_fhpush.js

```diff
@@ -26,6 +26,7 @@ describe('$fh.push', function () {
         widget: 'a',
         instance: 'b',
         millicore: 'fake-domain.feedhenry.com',
+        protocol: 'https',
```

Comment:
Maybe add a test that uses `http` just to make sure the code path work the same with http.